minecraft_servers/traidio/manifest.json

```diff
@@ -0,0 +1,102 @@
+{
+  "server_name": "traidio",
+  "nice_name": "TraidioNET",
+  "direct_ip": "traidio.net",
+  "server_wildcards": [
+    "%.traidio.net"
```

Comment:
Not a real wildcard
[https://i.imgur.com/hH25GpU.png](https://i.imgur.com/hH25GpU.png)

Reply:
![dns](https://github.com/LabyMod/server-media/assets/38141578/53fe113a-7514-4974-a122-37dbbccf3ee0)

Reply:
> A wildcard DNS record is a record in a DNS zone that will match requests for non-existent domain names.
As there's no redirect of my non-existent domain, it's not a wildcard.